### PR TITLE
logstalgia 1.1.3

### PR DESCRIPTION
**Pull request by chenrui333**
Created by `brew bump`

---

Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by cho-m** [comment]
Need to update `pcre` dependency to `pcre2`:
```
checking for PCRE2... configure: error: Package requirements (libpcre2-8) were not met:

No package 'libpcre2-8' found
```

**Comment by SMillerDev** [comment]
```
  configure: error: Could not link against -lGLU !
```

**Comment by chenrui333** [comment]
```
  Broken dependencies:
    /opt/homebrew/opt/boost/lib/libboost_filesystem.dylib (boost)
    /opt/homebrew/opt/boost/lib/libboost_system.dylib (boost)
```

**Comment by cho-m** [comment]
If boost is getting linked, then it may need to be a runtime dependency rather than just a build dependency

File: Formula/logstalgia.rb [hunk 1]
```diff
@@ -1,10 +1,9 @@
 class Logstalgia < Formula
   desc "Web server access log visualizer with retro style"
   homepage "https://logstalgia.io/"
-  url "https://github.com/acaudwell/Logstalgia/releases/download/logstalgia-1.1.2/logstalgia-1.1.2.tar.gz"
-  sha256 "ed3f4081e401f4a509761a7204bdbd7c34f8f1aff9dcb030348885fb3995fca9"
+  url "https://github.com/acaudwell/Logstalgia/releases/download/logstalgia-1.1.3/logstalgia-1.1.3.tar.gz"
+  sha256 "82e6a33c3c305c1f1d32d7e115ba0b307bb191ed2a70368a3cd9138ced0a98d9"
   license "GPL-3.0"
```
Comment (SMillerDev) [inline Formula/logstalgia.rb:6]:
```suggestion
  license "GPL-3.0-or-later"
```

**Comment by SMillerDev** [comment]
Doesn't seem to be failing anymore

**Review by cho-m — APPROVED** [review]

**Pull request closed without merging**